pipelines/DECam/ApPipe.yaml

```diff
@@ -48,6 +50,7 @@ subsets:
       - filterDiaSrcCat
       - rbClassify
       - transformDiaSrcCat
+      - mpSkyEphemerisQuery
```

Comment:
This task should _not_ be in `prompt` (more generally, `preload`, `prompt`, and `afterburner` should be disjoint).